server/controllers/adminController.js

```diff
@@ -134,6 +142,108 @@ class AdminController {
 
     return successStat(res, 200, 'data', foundUser);
   }
+
+  /**
+    * @static
+    * @description Allows a super-admin to create new user
+    * @param {Object} req - Request object
+    * @param {Object} res - Response object
+    * @returns {Object} object containing user data and access Token
+    * @memberof AdminController
+    */
+  static async createUser(req, res) {
+    const {
+      firstname, lastname, email, username
+    } = req.body.user;
+    const userRole = req.body.user.role;
+    const { role } = req.user;
+    if (role !== 'god') return errorStat(res, 401, 'You cannot perform this action. Please contact a god');
+    const password = process.env.NEW_USER_PASSWORD;
```

Comment:
I think the new user password should be automatically generated rather than having it as a static password for all user. Overall nice work

Reply:
Thank you for the observation. A random password would be great. I decided to make it simple thats why we have a static password. The user is notified in the mail sent to them to change their password.